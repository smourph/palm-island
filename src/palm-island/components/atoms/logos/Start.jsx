import React from 'react';
import start from '@internal/resources/images/logo-start-white.png';
import './Logo.scss';

export const Start = () => (
  <span className="logo logo-common logo-start">
    <img src={start} alt="start"/>
  </span>
);

export default Start;
