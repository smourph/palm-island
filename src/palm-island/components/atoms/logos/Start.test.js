import React from 'react';
import {render, screen} from '@testing-library/react';
import Start from './Start';

it('should render snapshot', () => {
  const {asFragment} = render(<Start/>);
  expect(asFragment().firstChild).toMatchSnapshot();
});

it('should render start', () => {
  render(<Start/>);

  const icon = screen.getByAltText('start');
  expect(icon).toBeInTheDocument();
  expect(icon).toBeVisible();
  expect(icon).toHaveAttribute('src', 'logo-start-white.png');
});
