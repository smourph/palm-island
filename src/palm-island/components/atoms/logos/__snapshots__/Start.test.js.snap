// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
<span
  class="logo logo-common logo-start"
>
  <img
    alt="start"
    src="logo-start-white.png"
  />
</span>
`;
